pkg/unittest/default_k8sclient.go

```diff
@@ -85,6 +85,10 @@ func (f *fakeK8sClient) CRDClient() k8scrdclient.Interface {
 	return nil
 }
 
+func (f *fakeK8sClient) CtrlCache() client.Reader {
+	return f.ctrlClient
```

Comment:
Cache implementation in `controller-runtime` doesn't have "fake" implementation. Kind of makes sense because the fake `client.Client` is in-memory already. Therefore this shares `CtrlClient()` with `CtrlCache()` to provide unified view.